determined-ai/determined
# feat: more welcoming home page in doc

## Description
[WEB-1357]




## Test Plan
- Check if home page looks more welcoming, and home page doesn't contain sidebar
- Check if tiles looks well aligned in different viewport sizes
- Check if content page is aligned center




## Commentary (optional)
- Did not include mobile view layout fix since it would require more investigation




## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-1357]: https://hpe-aiatscale.atlassian.net/browse/WEB-1357?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
